source/actions/get-images.js

```diff
@@ -40,6 +43,10 @@ export default async function (args, options, logger) {
     if (answers.namePattern) namePattern = answers.namePattern;
   }
 
+  pathToSave = pathValidator(pathToSave);
```

Comment:
A validate function should only return true/false or throw Error. Your way of writing & using `pathValidator` is more similar to a format function.

On the other hand, to name the functions, you should choose verb instead of noun, which should be used for Class/object.

In this case, if the purpose is just to validate the options and there's no value manipulation, you can use something like:
`validatePath()`, which will return nothing, and only throw Error if the input value is invalid. Usage: `validatePath(pathToSave)`